### PR TITLE
Improve transcription integration test and run as Github workflow action

### PR DESCRIPTION
**Pull request by noahmako**
**Issue #:** N/A.

**Description of changes:** This change improves the stability of the transcription integration test and integrate it into the miscellaneous integration test GitHub workflow.

**Testing:** Tests pass with local selenium grid.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.* The change can be confirmed by clicking on the "Pre-recorded speech" option in the demo app and observing that the audio file plays exactly once and does not loop.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No, this PR only touches integration tests & demo app functionality.

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No, this PR only touches integration tests & demo app functionality.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ziyiz-amzn — APPROVED** [review]

File: integration/configs/transcription_test.config.json [hunk 1]
```diff
@@ -10,13 +10,10 @@
         "noOfThreads": 1,
         "testImpl": "TranscriptionTest.js",
         "payload": {
-          "url": "http://localhost:8080/",
+          "url": "http://localhost:8080/?region=us-east-1",
+          "region": "us-east-1",
```
Comment (ltrung) [inline integration/configs/transcription_test.config.json:14]: Don't think you need this if you specify it in the URL already.

Comment (noahmako) [reply integration/configs/transcription_test.config.json:14]: The metrics get the region string from the payload rather than the query parameter.

Comment (ltrung) [reply integration/configs/transcription_test.config.json:14]: Then do you need the query parameter?

Comment (noahmako) [reply integration/configs/transcription_test.config.json:14]: Updated to only use the payload parameter and wired it up through the test.

File: .github/workflows/transcription-integration.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+  name: Transcription Integration Workflow
+
+  on:
+    pull_request:
+      branches:
+        - master
+        - release-1.x
```
Comment (ltrung) [inline .github/workflows/transcription-integration.yml:7]: You don't need to run on release-1.x branch

File: .github/workflows/transcription-integration.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    integ-data-message:
+      name: Transcription Integration Test
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout Package
+          uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+        - name: Pack the Chime SDK and install the tarball into the Demo
```
Comment (ltrung) [inline .github/workflows/transcription-integration.yml:26]: This should not be needed.

**Review by ltrung — APPROVED** [review]

**Pull request merged**